x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.test.js

```diff
@@ -280,7 +280,7 @@ describe('pluckStyleMetaFromSourceDataRequest', () => {
       );
 
       const featuresMeta = await vectorStyle.pluckStyleMetaFromSourceDataRequest(sourceDataRequest);
-      expect(featuresMeta.myDynamicField).toEqual({
+      expect(featuresMeta.fieldMeta.myDynamicField.range).toEqual({
```

Comment:
To be consistent, do can you rename `featuresMeta` to `styleMeta` like in VectorStyle?